mart/attack/adversary_in_art.py

```diff
@@ -21,7 +24,7 @@ class MartToArtAttackAdapter:
     but it should be reusable for other models with slight modifications.
     """
 
-    def __init__(self, target_model, mart_exp_config_yaml, **kwargs):
+    def __init__(self, target_model: torch.nn.Module, mart_exp_config_yaml: str, **kwargs):
```

Comment:
I think this class should adopt the types in https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/987052c405e05d458276299aafc7d47bb584e738/art/estimators/object_detection/object_detector.py#L42

Reply:
Should we use the ART's `ObjectDetector` type in the `target_model` argument?

Reply:
Currently, we expect it to be `ObjectDetector`. But later we may want to use `BaseEstimator` as we generalize the wrapper for non-object detection models in ART.